### ISSUE TITLE
Make unread mails bolder

### ISSUE THREAD

**Issue by frumble** [conema/monterail-dark#54]
Monterail Dark 2 for TB>68 is using a less bold unread mail font choice than the old one. It’s difficult to see which one are read an which one are unread.
Please choose a bolder font setting.

I’m experiencing this on Thunderbird 78.4.3, Monterail Dark 3.1, Arch Linux, KDE Plasma with standard Breeze theme.

![Screenshot_20201121_191241](https://user-images.githubusercontent.com/1595877/99884336-da25e880-2c2d-11eb-8c95-f2db0001bb01.png)

**Comment by conema** [comment]
Hi @frumble, the situation should be a bit better, could you try this fix?
[monterail-dark.zip](https://github.com/conema/monterail-dark/files/5582225/monterail-dark.zip)

**Comment by tsokar** [comment]
Hi !
I have the same problem, but the fix did have a weird side effect on the font which seems to be condensed now.
Before
![Screenshot_20201126_083600](https://user-images.githubusercontent.com/456434/100321582-32a90d00-2fc3-11eb-9b60-a2bbee4c2e70.png)

After
![Screenshot_20201126_083727](https://user-images.githubusercontent.com/456434/100321600-3b014800-2fc3-11eb-873b-ad10976c7982.png)

Thanks !

**Comment by conema** [comment]
Hi @tsokar,
Thank you for the screenshot.

![immagine](https://user-images.githubusercontent.com/12801153/100325531-15773d00-2fc9-11eb-837d-15d4666617d7.png)
Which one of the last two you prefer?
(The first one is how it look on Windows, but it's broke on linux, we need to change the font to fix the issue)

I give you also the updated theme, so you can see it live.
[monterail-dark.zip](https://github.com/conema/monterail-dark/files/5601443/monterail-dark.zip)

**Comment by tsokar** [comment]
Dear @conema , thanks a lot !
The last version you sent is perfect for me :-)

**Comment by conema** [comment]
@kionez @rnc
Sorry if I bother you, but I'd like to know also your opinions about the font change. (if you have the time)

[monterail-dark.zip](https://github.com/conema/monterail-dark/files/5601443/monterail-dark.zip)

**Comment by rnc** [comment]
Either of them is fine by me :-)

**Comment by kionez** [comment]
Either of them is fine also by me, I override the font in my userChrome.css for consistency with other linux apps.
I would like to test if "-moz-os-version" is useful to detect OS and change font only on some platform, but ATM I have no spare time.

**Comment by hubbty** [comment]
> @kionez @rnc @hubbty
> Sorry if I bother you, but I'd like to know also your opinions about the font change. (if you have the time)
> 
> [monterail-dark.zip](https://github.com/conema/monterail-dark/files/5601443/monterail-dark.zip)

This looks perfectly fine, just like the previous version did (I forgot to take a screenshot before I replaced it). Afterwards I downloaded "monterail_dark-1.2.5-tb" in order to compare it, but could not install that being told the file was corrupt (2 downloads).

Next I installed "monterail_dark_2_for_tb68-3.1.0-tb" and this looks to me exactly like the version linked above (only the settings options are missing).
